ci/build.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.14.2")
```

Comment:
This version got repeated vs. `ci-signing`.

Reply:
Probably possible to avoid. Just a bit trickier b/c of the self-hosting thing.

Reply:
It will probably never get out of sync in practice, especially if Steward manages it.  It's ... probably fine.